app/renderer/abr-document.js

```diff
@@ -207,6 +218,48 @@ function AbrDocument () {
             }
         };
 
+        // Handle ALT modifier key
+        var ShiftKeyHandler = function(e) {
+            var linkIsClickable = !that.cm.somethingSelected() &&
+                e.type === "keydown" &&
+                e.shiftKey;
+            document.body.classList.toggle("shift-pressed", linkIsClickable);
```

Comment:
This part is smart, but I had to read it twice to understand why you did it this way. Could you rename the class name in something less confusing (maybe "link-clickable")?